Cargo.toml

```diff
@@ -21,7 +21,7 @@ libc = "0.2"
 geo-types = { version = "0.7" }
 gdal-sys = { path = "gdal-sys", version = "^0.8" }
 ndarray = { version = "0.15", optional = true }
-chrono = { version = "0.4" }
+chrono = { version = "<=0.4.22" }
```

Comment:
This will be reverted when #335 is addressed.

Reply:
I'm not super fluent in my cargo versioning constraints, but should this be: `>= 0.4.0 <=0.4.22`